ffcx/codegeneration/lnodes.py

```diff
@@ -29,12 +29,12 @@
 from typing import List, Optional, Sequence
```

Comment:
Maybe these could be removed by pyupgrade upon bumping the minimum required python version (matching the one in dolfinx).